.airtap.yml

```diff
@@ -1,11 +0,0 @@
-providers:
```

Comment:
Haha I was hesitant to remove it in case we ever want to test this lib in browser. I think your call is a good call!